src/components/Drawer/Navigation/Navigation.style.ts

```diff
@@ -32,8 +32,31 @@ const itemStyle = (theme: Theme): SerializedStyles => css`
   cursor: default;
 `;
 
+const menuStateStyles = (theme: Theme): SerializedStyles => css`
+  &:hover {
+    background-color: ${getHover({ theme }).backgroundColor}; !important;
```

Comment:
Can you explain the logic of background-coloring here? I think that since every state has `!important` and it's not used to add arbitrary specificity to one state over the others, then it could be omitted altogether?